### PR TITLE
[JENKINS-54133] Encode console notes on master for JEP-210 compatibility

### PR DESCRIPTION
**Pull request by jglick**
[JENKINS-54133](https://issues.jenkins-ci.org/browse/JENKINS-54133): As opposed to the general case in https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/80, in this case it is easy enough to be compatible with JEP-210. Test deps are on https://github.com/jenkinsci/durable-task-plugin/pull/86 + https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/87.

### CODE REVIEW

**Comment by jglick** [comment]
There is a timeout in `AntWrapperTest.durability` which I can reproduce locally but do not yet understand. It does not appear to be caused by the use of watch mode. Apparently there are two similar `WorkflowRun`s after the restart, perhaps indicating yet another bug in `AbstractLazyLoadRunMap`?

**Comment by jglick** [comment]
Weirdness. With `workflow-durable-task-step` 2.16, the test passes. With 2.18, it fails. With 2.17, it passes, but without clearly wrong things happening, which I blame on https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/53. The failures in 2.18 seem to be due to https://github.com/jenkinsci/durable-task-plugin/pull/49 and yet, with https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/53 reverted by https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/81, the test can pass even with the current `durable-task` so long as watch mode (https://github.com/jenkinsci/durable-task-plugin/pull/60) is disabled. I do not really know what is going on yet.

File: pom.xml [hunk 1]
```diff
@@ -24,18 +24,21 @@
...
     <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
-    <tag>HEAD</tag>
+    <tag>${scmTag}</tag>
   </scm>
 
   <properties>
-    <jenkins.version>1.651.3</jenkins.version>
+    <revision>1.9</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jenkins.version>2.121.2</jenkins.version>
```
Comment (dwnusbaum) [inline pom.xml]: Big jump, but required for JEP-210 compatibility. Based on [these usage stats](http://stats.jenkins.io/pluginversions/ant.html), around 73% of users of ant 1.8 are on Jenkins 2.121.2 or newer anyway, so this seems fine.

Comment (jglick) [reply pom.xml]: Anyway even if there is some unrelated fix we realize we need to deliver urgently to the remaining 27%, it is easy enough to publish a backport.

**Pull request merged**